modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -43,33 +30,33 @@ apiVersion: operator.openshift.io/v1
 kind: Network
 metadata:
   name: cluster
-  spec:
+spec:
   additionalNetworks:
-  - name: test-network-1
-    namespace: additional-network-1
-    type: Raw
-    rawCNIConfig: '{
-      "cniVersion": "0.3.1",
-      "name": "macvlan-vrf",
-      "plugins": [  <1>
-      {
-        "type": "macvlan",  <2>
-        "master": "eth1",
-        "ipam": {
-            "type": "static",
-            "addresses": [
-            {
-                "address": "191.168.1.23/24"
-            }
-            ]
-        }
-      },
-      {
-        "type": "vrf",
-        "vrfname": "example-vrf-name",  <3>
-        "table": 1001   <4>
-      }]
-    }'
+    - name: test-network-1
+      namespace: additional-network-1
+      type: Raw
+      rawCNIConfig: '{
+        "cniVersion": "0.3.1",
+        "name": "macvlan-vrf",
+        "plugins": [  <1>
+        {
+          "type": "macvlan",  <2>
```

Comment:
A bit confused. The description here specifies that the type must be set to "vrf". I believe it should be pointed to the second item in the list.